### ISSUE TITLE
Your Add To Basket element is not accessible via the keyboard

### ISSUE THREAD

**Issue by EmmanuelDemey** [EmmanuelDemey/GlobalIssueTracker#36]
# Informations générales

* URL du site : https://www.thekase.com/fr/fr/iphone-xr-reconditionne-64-go-noir-sans-face-id-debloque-38972114.html

# Descriptions

People using the keyboard for browsing to the web (people with disability, are not able to add product to their baskets. Because your are using a link without a href. Please use a button instead. 

![image](https://user-images.githubusercontent.com/555768/216048179-a493c436-9198-43f0-bdcf-8a271aeb4712.png)

# Quelques Liens
